docs/source/basic/terms.rst

```diff
@@ -0,0 +1,70 @@
+Terms
+=====
+
+This page provides an overview of the terms used in ``alpaka`` and the relationships between them.
+
+Platform
+--------
+
+- A ``platform`` contains information about the system, e.g. the available devices. 
+- Depending on the platform, it also contains a runtime context.
+- A ``platform`` can be shared by many devices.
+
+Device
+------
+
+- A ``device`` represent a compute unit, such as a CPU or a GPU.
+- Each ``device`` is bounded to a specific ``platform``.
+- Each ``device`` can be used by many specific ``accelerators``.
+
+Work division
+-------------
+
+- Describes the domain decomposition of a contiguous N-dimensional index domain in ``blocks``, ``threads`` within a ``block``, and ``elements`` per ``thread``.
+- A ``work division`` has limitations depending on the ``kernel`` function and ``accelerator``.
+
+Accelerator
+-----------
+
+- Describes "how" a kernel work division is mapped to device threads.
+  - N-dimensional work divisions (1D, 2D, 3D) are supported.
+  - Holds implementations of shared memory, atomic operations, math operations etc.
+- ``Accelerators`` are instantiated only when a kernel is executed, and can only be accessed in device code.
+  - Each device function can (should) be templated on the accelerator type, and take an accelerator as its first argument.
```

Comment:
Looks like the indentation is not correct. It does not render as sub items.

Reply:
fixed